utils/build/docker/python/parametric/apm_test_client/server.py

```diff
@@ -145,11 +145,15 @@ def OtelStartSpan(self, request, context):
                 )
             )
 
+        # Python SpanKind enum starts at 0 whereas the proto spec used here starts at 1
+        # skin_kind = 0 is used for UNSPECIFIED in test library, we handle it as INTERNAL
+        span_kind = request.span_kind - 1 if request.span_kind > 0 else 0
```

Comment:
Nice catch but I do not like how this argument is defined. `UNSPECIFIED` is not a valid state for SpanKind. According to the spec we should only support internal, server, client, consumer, producer: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#spankind. 

If the SpanKind needs to be "unspecified" then we should change the type of this argument to an Optional[int].